samples/core/src/main/java/io/oasp/gastronomy/restaurant/salesmanagement/dataaccess/api/BillEntity.java

```diff
@@ -24,6 +24,7 @@
  */
 @Entity
 @Table(name = "Bill")
+// @Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
```

Comment:
Is the line kept commented for documentation purposes?

no? so remove it... 

yes? so, document it, please